### PR TITLE
Updated banner

### PR DESCRIPTION
**Pull request by jonathanbobel**
## Description

New banner text and link for IDP updates

### CODE REVIEW

File: _layouts/landing.html [hunk 1]
```diff
@@ -6,11 +6,12 @@
   <div
     class="bg-gold-20v padding-x-2 desktop:padding-x-4 padding-y-1 line-height-sans-3 font-lang-4 text-bold"
   >
-    USWDS 3.0 is here.
+    USWDS design patterns:
```
Comment (thisisdano) [inline _layouts/landing.html:9]:
```suggestion
    Introducing inclusive design patterns:
```

File: pages/home.md [hunk 2]
```diff
@@ -42,16 +42,16 @@ graphic_list:
...
       image:
         src: img/home/page-templates.svg
         alt:
         width: 72
         height: 72
       link:
-        text: Start from a template
-        url: page-templates
+        text: Explore pattern guidance
+        url: patterns
```
Comment (thisisdano) [inline pages/home.md:54]: Let's move this to position 2, after components

**Review by thisisdano — CHANGES_REQUESTED** [review]

**Comment by jonadecker** [comment]
Looks good to me!

**Review by thisisdano — APPROVED** [review]

**Comment by thisisdano** [comment]
Merging over Circle fails:
- `test_build`: Armypubs site is down

**Pull request merged**